gemma/gemma_args.h

```diff
@@ -28,13 +28,205 @@
 #include "compression/shared.h"
 #include "gemma/common.h"
 #include "gemma/gemma.h"  // For CreateGemma
+#include "hwy/base.h"     // HWY_IS_ASAN, HWY_ABORT
 #include "ops/matmul.h"
+#include "util/allocator.h"
 #include "util/args.h"
 #include "util/basics.h"  // Tristate
-#include "hwy/base.h"       // HWY_ABORT
+#include "util/threading.h"
+#include "util/threading_context.h"
 
 namespace gcpp {
 
+static inline const char* CompiledConfig() {
+  if (HWY_IS_ASAN) {
+    return "asan";
+  } else if (HWY_IS_MSAN) {
+    return "msan";
+  } else if (HWY_IS_TSAN) {
+    return "tsan";
+  } else if (HWY_IS_HWASAN) {
+    return "hwasan";
+  } else if (HWY_IS_UBSAN) {
+    return "ubsan";
+  } else if (HWY_IS_DEBUG_BUILD) {
+    return "dbg";
+  } else {
+    return "opt";
+  }
+}
+template <typename Derived>
+struct ArgsBase {
```

Comment:
hm, this was/still is defined in util/args.h, is it necessary to change the base class for adding the flag?
If you want to improve the ArgsBase, we can overwrite the one in util/args.h instead?